Documentation/TimeSlider/README.md

```diff
@@ -56,33 +56,33 @@ User can control whether to show `Slider` and/or `Playback Buttons` using `isSli
 Initialize TimeSlider and add constraints to position it.
 
 ```swift
-    // Configure time slider
-    let timeSlider = TimeSlider()
-    timeSlider.labelMode = .ticks
-    timeSlider.addTarget(self, action: #selector(TimeSliderExample.timeSliderValueChanged(timeSlider:)), for: .valueChanged)
-    view.addSubview(timeSlider)
-
-    // Add constraints to position the slider
-    let margin: CGFloat = 10.0
-    timeSlider.translatesAutoresizingMaskIntoConstraints = false
-    timeSlider.bottomAnchor.constraint(equalTo: mapView.attributionTopAnchor, constant: -margin).isActive = true
-
-    if #available(iOS 11.0, *) {
-        timeSlider.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor, constant: margin).isActive = true
-        timeSlider.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor, constant: -margin).isActive = true
-    }
-    else {
-        timeSlider.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: margin).isActive = true
-        timeSlider.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -margin).isActive = true
-    }
+// Configure time slider
+let timeSlider = TimeSlider()
+timeSlider.labelMode = .ticks
+timeSlider.addTarget(self, action: #selector(TimeSliderExample.timeSliderValueChanged(timeSlider:)), for: .valueChanged)
+view.addSubview(timeSlider)
+
+// Add constraints to position the slider
+let margin: CGFloat = 10.0
+timeSlider.translatesAutoresizingMaskIntoConstraints = false
+timeSlider.bottomAnchor.constraint(equalTo: mapView.attributionTopAnchor, constant: -margin).isActive = true
+
+if #available(iOS 11.0, *) {
```

Comment:
IMO with the minimum required Runtime SDK version as 100.15.0, the min iOS version should be iOS 13, so this API check can be removed?

Reply:
Nice catch!